src/react/hooks/use-typing.ts

```diff
@@ -92,14 +92,13 @@ export const useTyping = (params?: TypingParams): UseTypingResponse => {
 
     void context.room
       .then((room) => {
-        if (!mounted) return;
-
         // If we're not attached, we can't call typing.get() right now
         if (room.status === RoomStatus.Attached) {
           const typing = room.typing.get();
           logger.debug('useTyping(); room attached, getting initial typers', { typing });
           setCurrentlyTyping(typing);
         } else {
+          if (!mounted) return;
```

Comment:
Was this move intentional? Do we want to call `setCurrentlyTyping` even if we're not still mounted?

Reply:
Nope, the code block is now redundant anyway, so removed in 5f8556bad0897b3fbf0561b094964188f92185dc